composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "minimum-stability": "dev",
     "require": {
-        "php": "^7.1.3",
+        "php": "^7.2.5",
```

Comment:
Playing devils advocate - How would this affect support for Symfony 3.4 && why not bump to 7.3.x as 7.2 ends security support at the end of November?

Reply:
Its not needed, 7.2.5 is the minimum version for symfony string 👍